sw/device/lib/testing/uart_testutils.h

```diff
@@ -0,0 +1,47 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_TESTING_UART_TESTUTILS_H_
+#define OPENTITAN_SW_DEVICE_LIB_TESTING_UART_TESTUTILS_H_
+
+#include <stdint.h>
+
+#include "sw/device/lib/base/status.h"
+#include "sw/device/lib/dif/dif_pinmux.h"
+#include "sw/device/lib/dif/dif_uart.h"
+
+/**
+ * Define the available platforms which uart is mapped
+ */
+typedef enum uart_pinmux_platform_id {
+  UartPinmuxPlatformIdHyper310 = 0,
```

Comment:
Is this `= 0` needed to follow a style guide? I think C's enum will always set the first entry to zero unless otherwise directed?

Reply:
I copied this style from another file, but am not actually sure what the style guide for it is.

I [found this](https://google.github.io/styleguide/cppguide.html#Enumerator_Names) code snippet in the Google style guide:

```c
enum class UrlTableError {
  kOk = 0,
  kOutOfMemory,
  kMalformedInput,
};
```

which uses this style, but don't see a written rule that you should make the first assignment explicit anywhere in here.